src/app/services/graph.service.js

```diff
@@ -301,12 +308,18 @@ angular
                 el.data['selected'] = 1;
             }
 
-            if (el.data.docs && el.data.docs.show === false) {
+            // a model can be hidden by the configuration under docs or under config.docs
+            if (! ( _.get(el,['data', 'docs','show'],true)) ) {
                 el.data['hidden'] = 1;
             }
+
+            var color_config = _.get(el, ['data', 'config', 'docs', 'node_color'])
```

Comment:
@b-per Is this still expected? If I remove the 'config' portion in the list, the custom colors do NOT work.

Reply:
I just did a test with the latest dbt-core and it works without looking at `config`. I think that we said we'd want to read from docs directly so I will push a small change.